src/Raven.Client/ServerWide/Operations/TrafficWatch/GetTrafficWatchConfigurationOperation.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using Raven.Client.Documents.Changes;
+using Raven.Client.Documents.Conventions;
+using Raven.Client.Http;
+using Raven.Client.Json.Serialization;
+using Sparrow.Json;
+using Size = Sparrow.Size;
+
+namespace Raven.Client.ServerWide.Operations.TrafficWatch;
+
+public class GetTrafficWatchConfigurationOperation : IServerOperation<TrafficWatchConfigurationResult>
+{
+    public RavenCommand<TrafficWatchConfigurationResult> GetCommand(DocumentConventions conventions, JsonOperationContext context)
+    {
+        return new GetTrafficWatchConfigurationCommand();
+    }
+
+    public class GetTrafficWatchConfigurationCommand : RavenCommand<TrafficWatchConfigurationResult>
+    {
+        public override bool IsReadRequest => true;
+        public override HttpRequestMessage CreateRequest(JsonOperationContext ctx, ServerNode node, out string url)
+        {
+            url = $"{node.Url}/admin/traffic-watch/configuration";
+
+            return new HttpRequestMessage(HttpMethod.Get, url);
+        }
+
+        public override void SetResponse(JsonOperationContext context, BlittableJsonReaderObject response, bool fromCache)
+        {
+            if (response == null)
+                ThrowInvalidResponse();
+
+            Result = JsonDeserializationClient.GetTrafficWatchConfigurationResult(response);
+        }
+    }
+}
+
+public class TrafficWatchConfigurationResult
+{
+    /// <summary>
+    /// Traffic Watch logging mode.
+    /// </summary>
+    public TrafficWatchMode TrafficWatchMode { get; set; }
+
+    /// <summary>
+    /// Database names by which the Traffic Watch logging entities will be filtered.
+    /// </summary>
+    public HashSet<string> Databases { get; set; }
+
+    /// <summary>
+    /// Response status codes by which the Traffic Watch logging entities will be filtered.
+    /// </summary>
+    public HashSet<int> StatusCodes { get; set; }
```

Comment:
what if we change that to strings and support things like: 4*, 5*, or '500', '200' ?

Reply:
there are 28 http status codes which starts with '4'

Reply:
We aren't using a lot of status codes.
If you are talking about auto-complete in the studio, common status codes should be available at the top (200, 201, 500, 503, 304, 403, 404, 409) and the others below that